src/main/java/com/google/jenkins/plugins/cloudbuild/client/ClientFactory.java

```diff
@@ -61,11 +61,18 @@ public ClientFactory(Run<?, ?> run, TaskListener listener, String credentialsId)
 
     CloudBuildScopeRequirement requirement = new CloudBuildScopeRequirement();
     this.credentials = CredentialsProvider.findCredentialById(
-        credentialsId, GoogleRobotPrivateKeyCredentials.class, run, requirement);
+        credentialsId, GoogleRobotCredentials.class, run, requirement);
     if (credentials == null) {
       throw new AbortException(Messages.ClientFactory_FailedToRetrieveCredentials(credentialsId));
     }
-    this.gcred = credentials.getGoogleCredential(requirement);
+    try {
+      this.gcred = credentials.getGoogleCredential(requirement);
+    } catch (GeneralSecurityException e) {
```

Comment:
Though `GoogleRobotPrivateKeyCredentais#getGoogleCredential()` throws only exceptions derived from `RuntimeException`, 
the base class `GoogleRobotCredentials#getGoogleCredential()` (actually `GoogleOAuth2Credentials#getGoogleCredential()` throws exceptions not derived from `RuntimeException`.
* http://javadoc.jenkins.io/plugin/google-oauth-plugin/com/google/jenkins/plugins/credentials/oauth/GoogleRobotPrivateKeyCredentials.html#getGoogleCredential-com.google.jenkins.plugins.credentials.oauth.GoogleOAuth2ScopeRequirement-
* http://javadoc.jenkins.io/plugin/google-oauth-plugin/com/google/jenkins/plugins/credentials/oauth/GoogleOAuth2Credentials.html#getGoogleCredential-com.google.jenkins.plugins.credentials.oauth.GoogleOAuth2ScopeRequirement-